client/pages/stats.jsx

```diff
@@ -125,14 +126,40 @@ export default class Stats extends React.Component {
         </ol>
       );
     });
+
     return (
       <>
         {dates}
       </>
     );
   }
 
+  makePieChartData(pieChartDataArray) {
+    const data = [];
+    for (let i = 0; i < pieChartDataArray.length; i++) {
+      data.push(pieChartDataArray[i].total.toFixed(2));
+    }
+    return data;
+  }
```

Comment:
When you are looping and pushing like this, that means `.map()` is your go-to method.

```js
return pieChartDataArray.map(datum => datum.total.toFixed(2))
```